new_is_always_better.py

```diff
@@ -0,0 +1,81 @@
+import struct
+from typing import Dict
+from scapy.all import conf, IFACES, get_if_hwaddr
+import argparse
+from struct import unpack
```

Comment:
Nitpicking on my side - it is a convention to first import builtin libraries, and after that import external libraries (such as scapy). Also - it is common to arrange the rows based on length in ascending order.

```suggestion
import struct
import argparse
from typing import Dict
from struct import unpack

from scapy.all import conf, IFACES, get_if_hwaddr
```

Reply:
fixed